translation.json

```diff
@@ -0,0 +1,56 @@
+{
+  "widgets": {
+    "login": {
+      "dataSource": {
+        "login": {
+          "instruction": "Enter your access details below",
+          "emailPlaceholder": "Enter your email",
+          "passwordPlaceholder": "Enter your password",
+          "mismatch": "The details you entered don’t match our records. Please try again.",
+          "action": "Log in",
+          "forgotPassword": "Forgot your password?"
+        },
+        "restore": {
+          "instruction": "To reset your password you must verify your email address.",
+          "emailNotFound": "Your email address cannot be found. Please try again.",
+          "back": "Back to Login",
```

Comment:
You need to add the line `"emailMismatch": "Please enter a valid email address and try again.",`

Reply:
@OleksandrLavrynenko Added. Thanks.